apps/web/vibes/soul/examples/pages/error/index.tsx

```diff
@@ -0,0 +1,87 @@
+import { locales } from '@/vibes/soul/data/locales';
+import { action } from '@/vibes/soul/examples/primitives/inline-email-form/actions';
+import { localeAction } from '@/vibes/soul/examples/primitives/navigation/actions';
+import { logo, navigationLinks } from '@/vibes/soul/examples/primitives/navigation/electric';
+import { copyright, footerLinks } from '@/vibes/soul/examples/sections/footer/electric';
+import { Banner } from '@/vibes/soul/primitives/banner';
+import { Navigation } from '@/vibes/soul/primitives/navigation';
+import { Error } from '@/vibes/soul/sections/error';
+import { Footer } from '@/vibes/soul/sections/footer';
+import {
+  Amex,
+  ApplePay,
+  Bitcoin,
+  GooglePay,
+  Mastercard,
+  Paypal,
+  Visa,
+} from '@/vibes/soul/sections/footer/payment-icons';
+import { Facebook, Instagram, X, Youtube } from '@/vibes/soul/sections/footer/social-icons';
+import { Subscribe } from '@/vibes/soul/sections/subscribe';
+
+const socialMediaLinks = [
+  {
+    href: '#',
+    icon: <Facebook />,
+  },
+  {
+    href: '#',
+    icon: <X />,
+  },
+  {
+    href: '#',
+    icon: <Instagram />,
+  },
+  {
+    href: '#',
+    icon: <Youtube />,
+  },
+];
+
+const paymentIconsArray: React.ReactNode[] = [
+  <Visa key="Visa" />,
+  <Amex key="Amex" />,
+  <Mastercard key="Mastercard" />,
+  <Paypal key="Paypal" />,
+  <GooglePay key="GooglePay" />,
+  <ApplePay key="ApplePay" />,
+  <Bitcoin key="Bitcoin" />,
+];
+
+export default function Preview() {
+  return (
+    <>
+      <Banner id="example-banner">
+        Get <strong>15% off</strong> and free shipping with discount code{' '}
+        <strong>&quot;welcome&quot;</strong>
+      </Banner>
+
+      <Navigation
+        accountHref="#"
+        activeLocaleId="en"
+        cartHref="#"
+        links={navigationLinks}
+        localeAction={localeAction}
+        locales={locales}
+        logo={logo}
+        searchHref="#"
+      />
+
+      <Error cta={{ label: 'Try again', href: '#' }} />
```

Comment:
Since we already have default prop data included in the Error component, does it make sense to add the CTA prop data there as well so it's all in one place?

Reply:
@andrewreifman I was thinking the same thing at first, but I'm not privy to the idea of defaulting to `#` href that does nothing 🤔  The way it is currently, if `cta` prop is not provided at all, the button won't render. Otherwise, it will always use the default button, and the user would have to explicitly provide `undefined` if they don't want the button to show.